src/python/fuzzing/tests.py

```diff
@@ -903,9 +903,6 @@ def check_for_bad_build(job_type, crash_revision):
   if not environment.get_value('BAD_BUILD_CHECK'):
     return False
 
-  # Do not detect leaks while checking for bad builds.
-  environment.reset_current_memory_tool_options(leaks=False)
```

Comment:
I think disabling leak detection does not make sense, LSAN=False by default. And in jobs with LSAN=true, i think we care about leaks being found during bad build check.